src/parser/jobs/rdm/Combos.js

```diff
@@ -0,0 +1,112 @@
+import CoreCombos from 'parser/core/modules/Combos'
+import ACTIONS from 'data/ACTIONS'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import {Plural, Trans} from '@lingui/react'
+
+export default class Combos extends CoreCombos {
+	// Overrides
+	static suggestionIcon = ACTIONS.ENCHANTED_REDOUBLEMENT.icon
+
+	//These actions are considered a combo DERP
+	_derpComboActions = [
```

Comment:
I'd generally advise to keep this sort of meta outside the class itself - you can put variables in root scope, they get scoped to the current file.

Reply:
are you referencing the static suggestIcon or the _derpComboActions?  The first is done to override what's in the base class should I allow the base suggestion to appear.  Is there a functional benefit to keeping it outside the class?  I generally speaking will only put values outside the class that are truly static/const, since I'm modifying this array at runtime I didn't put it outside the class.